src/pages/settings/company/documents/components/Table.tsx

```diff
@@ -37,11 +37,12 @@ export function Table() {
   const [t] = useTranslation();
   const [currentPage, setCurrentPage] = useState<number>(1);
   const [perPage, setPerPage] = useState<string>('10');
+  const companyDocuments = 'true';
   const { dateFormat } = useCurrentCompanyDateFormats();
   const [document, setDocument] = useState('');
   const [isPasswordConfirmModalOpen, setPasswordConfirmModalOpen] =
     useState(false);
-  const { data, isLoading } = useDocumentsQuery({ perPage, currentPage });
+  const { data, isLoading } = useDocumentsQuery({ perPage, currentPage, companyDocuments });
```

Comment:
Since the variable never changes, just pass the direct value:

```ts
const { data, isLoading } = useDocumentsQuery({ perPage, currentPage, companyDocuments: 'false' });
```